.github/workflows/check-outdated-content.yml

```
@@ -0,0 +1,164 @@
+# Contribution by Cloud Native Guatemala folks :)
+# This workflow will check if a localized content is outdated or not
+# by comparing English content in the old branch and the latest branch.
+---
+name: Check outdated content
+
+on:
+#  pull_request:
```

```suggestion
  pull_request:
```

I guess we still want to keep pull_request option so that translation automation triggers as part of PRs but we still get an option to run it manually, on-demand.

---

Changed done @guidemetothemoon